client/src/components/layout/Alert.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Alert as AntAlert } from 'antd';
+
+const Alert = ({ alerts }) => {
```

Comment:
🔮 Is there any way we could make the Alert hover on the page instead of bumping the rest of it down when it appears?

Reply:
Probs with absolute positioning. Potentially the user could the  cover everything up if they trigger enough alerts, though

Reply:
We can do this in a follow-up. I'll file an issue.

Reply:
https://github.com/joshprzybyszewski/cribbage/issues/61

Reply:
Ope, just did it :#